### PR TITLE
chore: fix example comment at ParseParamComment func

### PR DESCRIPTION
**Pull request by Cloudzp**
**Describe the PR**
when use `swag init --parserVendor=true ` with pkg "github.com\swaggo\swag" in you vendor, will get an error:  ParseComment error in file E:\workspace\src\park_record\vendor\github.com\swaggo\swag\operation.go :form is not supported paramType

**Relation issue**
#422

### CODE REVIEW

File: operation.go [hunk 1]
```diff
@@ -118,7 +118,7 @@ func (operation *Operation) ParseComment(comment string, astFile *ast.File) erro
 var paramPattern = regexp.MustCompile(`(\S+)[\s]+([\w]+)[\s]+([\S.]+)[\s]+([\w]+)[\s]+"([^"]+)"`)
 
 // ParseParamComment parses params return []string of param properties
-// E.g. @Param	queryText		form	      string	  true		        "The email for login"
+// E.g. @Param	queryText		formData	      string	  true		        "The email for login"
```
Comment (easonlin404) [inline operation.go]: why this change can address #422 ?

Comment (Cloudzp) [reply operation.go]: paramType 'form' is not supported, but in you sample give a paramType "form", this can result `swag init` failed with error "paramType form not supported"

**Comment by easonlin404** [comment]
@Cloudzp Thanks for your contributions.
